js/pages/characterizations/components/feature-analyses/feature-analysis-view-edit.less

```diff
@@ -110,4 +110,15 @@
     font-weight: 500;
     margin-bottom: 1rem;
   }
+}
+
+
+.fe-design-disabled {
+  pointer-events: none;
+  opacity: 0.7;
```

Comment:
It's better to remove opacity from whole component since this makes harder to read large analyses.
Let's just prevent event handling and opacity for buttons.